internal/ceb/ssh.go

```diff
@@ -0,0 +1,263 @@
+package ceb
+
+import (
+	"context"
+	"crypto/x509"
+	"encoding/base64"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"strconv"
+	"syscall"
+
+	"github.com/creack/pty"
+	"github.com/gliderlabs/ssh"
+	"github.com/hashicorp/go-hclog"
+	"github.com/pkg/errors"
+	gossh "golang.org/x/crypto/ssh"
+)
+
+func RunExecSSHServer(ctx context.Context, logger hclog.Logger, sport, hostkey, key string) error {
```

Comment:
Is this supposed to be exported or expected to be called outside this package? I feel this should be unexported.

Reply:
It's called by the entrypoint so it has to be exported.